virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -38,6 +38,9 @@ The *Virtualization* section of the {product-title} web console contains the fol
 
 |xref:../../virt/getting_started/virt-web-console-overview.adoc#migrationpolicies-page_virt-web-console-overview[*MigrationPolicies* page]
 |Create and manage migration policies for workloads.
+
+|*Checkups* page
+|Run network latency and storage checkups for virtual machines.
```

Comment:
It's probably not in the scope of this PR, but this is the only page in the table without a link to more information.